File: Archiver/frmOutlookNotice.cs
using System;
using Microsoft.VisualBasic;
using MODI;

namespace EcmArchiver
{
    public partial class frmOutlookNotice
    {
        public frmOutlookNotice()
        {
            InitializeComponent();
            _btnTerminate.Name = "btnTerminate";
        }

        private clsUtility UTIL = new clsUtility();
        private int iSec = 1;

        private void btnTerminate_Click(object sender, EventArgs e)
        {
            UTIL.KillOutlookRunning();
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            statElapsedTime.Text = DateAndTime.TimeOfDay.Second.ToString();
            if (iSec > 60)
            {
                iSec = 1;
            }

            PB.Value = iSec;
        }

        private void frmOutlookNotice_Load(object sender, EventArgs e)
        {
            PB.Minimum = 0;
            PB.Maximum = 60;
        }
    }
}

File: Archiver/frmOutlookNotice.vb
Public Class frmOutlookNotice

    Dim UTIL As New clsUtility

    Dim iSec As Integer = 1

    Private Sub btnTerminate_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnTerminate.Click
        util.KillOutlookRunning()
    End Sub

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        statElapsedTime.Text = TimeOfDay.Second.ToString
        If iSec > 60 Then
            iSec = 1
        End If
        PB.Value = iSec
    End Sub

    Private Sub frmOutlookNotice_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        PB.Minimum = 0
        PB.Maximum = 60
    End Sub

End Class
